test/tools/ossfuzz/strictasm_assembly_ossfuzz.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <libyul/AssemblyStack.h>
+#include <libyul/backends/evm/EVMCodeTransform.h>
+
+using namespace yul;
+using namespace std;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* _data, size_t _size)
```

Comment:
```suggestion
extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
```

Reply:
`_data` is not a const pointer but a pointer to a const array of bytes.

Reply:
Yes indeed. Otherwise it would be `uint8_t * const _data`. The order of type and const-specifier is irrelevant for C/C++ and our coding style is "const-specifier after type".

Reply:
This always confuses me :) will fix.